addons/account/account_move_line.py

```diff
@@ -83,19 +83,23 @@ def _query_get(self, cr, uid, obj='l', context=None):
                 context['periods'] = fiscalperiod_obj.build_ctx_periods(cr, uid, first_period, context['period_from'])
             else:
                 context['periods'] = fiscalperiod_obj.build_ctx_periods(cr, uid, context['period_from'], context['period_to'])
+        if 'periods_special' in context:
+            periods_special = ' AND special = %s ' % bool(context.get('periods_special'))
+        else:
+            periods_special = ''
```

Comment:
Change this behavior must set a change on reports also because if not open reports will be behave differently than before... THANKS for that in a closing period date.....! #fail.

This is the reason because REPORTS MUST BE PART OF THE CORE UNITTEST ENVIRONMENT VALIDATING DATA OUTPUT.